### PR TITLE
LPS-148672

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi,

I have removed the deprecated in PropsKey.
Thanks for checking. Also see https://github.com/jeffwu0724/liferay-portal/pull/42.

Best,
Jeff

### CODE REVIEW

**Comment by jeffwu0724** [comment]
@kyle-miho Can you help me look at the test result, please?

Thanks
Jeff

**Comment by jeffwu0724** [comment]
@kyle-miho Can you help me look at the test result, please?

Thanks
Jeff

File: portal-kernel/src/com/liferay/portal/kernel/util/PropsKeys.java [hunk 1]
```diff
@@ -305,19 +305,6 @@
 		BLOGS_ENTRY_PREVIOUS_AND_NEXT_NAVIGATION_ENABLED =
 			"blogs.entry.previous.and.next.navigation.enabled";
 
-	/**
-	 * @deprecated As of Mueller (7.2.x), with no direct replacement
-	 */
-	@Deprecated
-	public static final String BLOGS_IMAGE_EXTENSIONS =
-		"blogs.image.extensions";
```
Comment (tinatian) [inline portal-kernel/src/com/liferay/portal/kernel/util/PropsKeys.java:320]: @jeffwu0724 Please search "blogs.image.extensions" in VerifyProperties, you would see "blogs.image.extensions" was already added to _MODULARIZED_PORTAL_KEYS. In this case, you do not need to add it to _OBSOLETE_PORTAL_KEYS.

Please check all other properties you removed, if it is same case as "blogs.image.extensions", please remove your changes for it in VerifyProperties

**Comment by tinatian** [comment]
@jeffwu0724 Please see my comment. I am going to close this pull, please resend.

Thanks.
Tina.

**Comment by tinatian** [comment]
ci:close

**Pull request closed without merging**